packages/teams-js/src/public/authentication.ts

```diff
@@ -328,6 +328,7 @@ export namespace authentication {
     } finally {
       Communication.childWindow = null;
       Communication.childOrigin = null;
+      GlobalVars.webAuthWindowOpen = false;
```

Comment:
As far as I can tell, every time the auth window is closed by TeamsJS it is done using this function. Because of that, I am using this as a bottleneck to set `webAuthWindowOpen` back to false.